### DISCUSSION TITLE
Flask Bundle remove white space after variables enclosed in ${} in Template strings in JavaScript.

### DISCUSSION THREAD

**Discussion by Rongeegee** [pallets/flask#5159] [Q&A]
So in my production environment, i use **Bundle** imported from **flask_assets**.

```
from flask_assets import Environment, Bundle

JS_BUNDLE = Bundle('js/modules/xxxxx.js',......,'js/modules/xxxxx.js',
                   filters=filters, output='js/bundle/scripts.custom.%s.min.js' % RELEASE_VERSION)

assets = Environment(app)
assets.register('js_custom', JS_BUNDLE)
```

Then in my JavaScript file, I have the following

```
const date = new Date();

const day = date.getDate() >= 10 ? date.getDate() :  "0" + date.getDate() ;
const month = date.toLocaleString('default', { month: 'long' }).substring(0,3);
const year = date.getFullYear();

const englishDate = `${day} ${month}, ${year}`;
```

Suppose that today's date is 6/8/2023. The englishDate is supposed to have a string value of "`08 Jun, 2023`". However, the value it got on production after the bundle is "`08Jun,2023`". All the spaces within the string is removed. 

I checked the bundled file, it turned out that the spaces are remove in the code when being bundled. The last line became 
```
const englishDate = `${day}${month},${year}`;
```
However, If I don't use variables inside the string enclosed by backticks (`). Such as 
```
const englishDate = `June 8th,  2023`;
```
All the spaces in the string will be preserved.

It seems like this issue only happened when a variable is used within a template string. All the spaces followed by the variable will be removed by the bundler.

Can anyone replicate it and let me know?

**Comment by ThiefMaster** [comment]
Sounds like a question for flask-assets, not for flask...
